src/version/version.rs

```diff
@@ -116,7 +142,11 @@ pub fn background_check_for_updates() -> mpsc::Receiver<Version> {
 
     let _detached_thread = thread::spawn(move || match check_wrangler_versions() {
         Ok(wrangler_versions) => {
-            if wrangler_versions.current != wrangler_versions.latest {
+            // If the wrangler version has not been checked within the last day and the versions
+            // are different, print out an update message
+            if !wrangler_versions.is_outdated()
+                && (wrangler_versions.current != wrangler_versions.latest)
```

Comment:
sorry, i  mean this whole big boolean:

``` rust
fn is_outdated(&self) -> bool {
    !self.checked && (self.current != self.latest)
}

# and then call
if wrangler_versions.is_outdated() {
    ...
}
```

Reply:
yeah this should make it look pretty clean